descwl_coadd/coadd.py

```diff
@@ -413,16 +413,19 @@ def make_coadd_old(
 
 
 def make_warps(
-    exp, coadd_wcs, coadd_bbox, psf_dims, rng, remove_poisson, max_maskfrac,
+    exp, coadd_wcs, coadd_bbox, psf_dims, rng, remove_poisson,
+    warper=None, mfrac_warper=None,
```

Comment:
what is the motivation to send the warper here?

Reply:
The `warper` has the warping kernels which should be configurable from the Task rather being hardcoded in here. This gives an option to override the defaults.